cmd/worker/main.go

```diff
@@ -2,18 +2,27 @@ package main
 
 import (
 	"log"
+	"os"
 
+	"github.com/beam-cloud/beta9/pkg/types"
 	"github.com/beam-cloud/beta9/pkg/worker"
 )
 
+func handleWorkerError(err error, context string) {
```

Comment:
hm do we really need a function for this?

Reply:
No not at all. I was on the fence on a copy/paste job or a func haha. I can just check the error twice.

Reply:
Updated.